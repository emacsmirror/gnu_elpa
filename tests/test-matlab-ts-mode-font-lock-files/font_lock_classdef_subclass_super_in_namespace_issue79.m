% -*- matlab-ts -*-

% See https://github.com/acristoffers/tree-sitter-matlab/issues/79

classdef font_lock_classdef_subclass_super_in_namespace_issue79 < myUtils.super
    methods
        function obj=SubClass
            obj@myUtils.super;
            disp('in SubClass')
        end
    end
end
