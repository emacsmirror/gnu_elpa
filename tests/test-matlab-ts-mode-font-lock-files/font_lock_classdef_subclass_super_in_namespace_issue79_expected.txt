c ccc ccccccccc ccc

c ccc cccccccccccccccccccccccccccccccccccccccccccccccccccccccccccc

kkkkkkkk ffffffffffffffffffffffffffffffffffffffffffffffffffffff o fffffffffffff
    kkkkkkk
        kkkkkkkk vvvoffffffff
            dddodddddddDdddddD
            BBBBbSss ssssssssSb
        kkk
    kkk
kkk
